stdlib/3/asyncio/tasks.pyi

```diff
@@ -96,9 +96,19 @@ class Task(Future[_T], Generic[_T]):
     def current_task(cls, loop: Optional[AbstractEventLoop] = ...) -> Task[Any]: ...
     @classmethod
     def all_tasks(cls, loop: Optional[AbstractEventLoop] = ...) -> Set[Task[Any]]: ...
-    def __init__(self, coro: Union[Generator[Any, None, _T], Awaitable[_T]], *, loop: AbstractEventLoop = ...) -> None: ...
+    if sys.version_info >= (3, 8):
+        def __init__(
+            self,
+            coro: Union[Generator[Any, None, _T], Awaitable[_T]],
+            *,
+            loop: AbstractEventLoop = ...,
+            name: Optional[str] = ...,
+        ) -> None: ...
+    else:
+        def __init__(self, coro: Union[Generator[Any, None, _T], Awaitable[_T]], *, loop: AbstractEventLoop = ...) -> None: ...
     def __repr__(self) -> str: ...
     if sys.version_info >= (3, 8):
+        def get_coro(self) -> Task[_T]: ...
```

Comment:
This actually returns the underlying coroutine, so strictly it should be `Union[Generator[Any, None, _T], Awaitable[_T]]`. Maybe we should fall back to Any, or maybe this is low-level enough that it pays to be strict.

Reply:
I guess I misread the "wrapped by" as "wrapped in" in the documentation. I'll go with `Any` for now and leave my obligatory remark that python/typing#566 could help.